Formula/nushell.rb

```diff
@@ -20,10 +20,6 @@ def install
   end
 
   test do
-    # assert_equal "#{Dir.pwd}> 2\n#{Dir.pwd}> CTRL-D\n", pipe_output("#{bin}/nu", 'echo \'{"foo":1, "bar":2}\' | from-json | get bar | echo $it')
-
-    # Remove the test below and return to the better one above if/when Nushell
-    # reinstates the expected behavior for Ctrl+D (EOF)
-    assert_match version.to_s, shell_output("#{bin}/nu --version")
+    assert_equal "#{Dir.pwd}> 2\n#{Dir.pwd}> CTRL-D\n", pipe_output("#{bin}/nu", 'echo \'{"foo":1, "bar":2}\' | from-json | get bar | echo $it')
```

Comment:
Please remove `CTRL-D\n` from the end of the assertion text here (keeping the space before it).